README.md

```diff
@@ -1,17 +1,141 @@
-# Project
+# GitHub Action for deploying to Azure Spring Cloud
+GitHub Actions support an automated software development lifecycle workflow. With GitHub Actions for Azure Spring Cloud you can create workflows in your repository to manage your deployment of Azure Spring Cloud conveniently.
 
-> This repo has been populated by an initial template to help get you started. Please
-> make sure to update the content to build a great experience for community-building.
+[comment]: <> (This repository contains GitHub Action for Azure Spring Cloud to deploy. The action supports deploying *.jar, *.war, and *.zip files, or a folder.)
 
-As the maintainer of this project, please make a few updates:
+[comment]: <> (As the maintainer of this project, please make a few updates:)
 
-- Improving this README.MD file to provide a great experience
-- Updating SUPPORT.MD with content about this project's support experience
-- Understanding the security reporting process in SECURITY.MD
-- Remove this section from the README
+[comment]: <> (- Improving this README.MD file to provide a great experience)
 
-## Contributing
+[comment]: <> (- Updating SUPPORT.MD with content about this project's support experience)
+
+[comment]: <> (- Understanding the security reporting process in SECURITY.MD)
+
+[comment]: <> (- Remove this section from the README)
```

Comment:
I presume `comment` won't be displayed in the final output. Shall we remove them? or remove the `comment` tag if the line actually provides some useful information?

Reply:
Sorry, latest version not saved, comments need to be deleted.